packages/migrate/index.ts

```diff
@@ -1,18 +1,31 @@
-"use strict";
+import chalk from "chalk";
+import * as diff from "diff";
+import * as fs from "fs";
+import * as inquirer from "inquirer";
+import * as Listr from "listr";
+import pLazy = require("p-lazy");
+import * as path from "path";
 
-const fs = require("fs");
-const path = require("path");
-const chalk = require("chalk");
-const diff = require("diff");
-const inquirer = require("inquirer");
-const PLazy = require("p-lazy");
-const Listr = require("listr");
+import * as runPrettier from "@webpack-cli/utils/run-prettier";
+
+import { transformations } from "./migrate";
+import { IJSCodeshift, INode } from "./types/NodePath";
+
+declare var process: {
+	cwd: Function;
+	webpackModule: {
+		validate: Function;
+		WebpackOptionsValidationError: any;
+	};
+	stdout: {
+		write: Function;
+	};
+	exitCode: number;
+};
 
 const { validate } = process.webpackModule;
```

Comment:
this needs to change to using `require('webpack'), bad rebase?

Reply:
This is same as `master`.

https://github.com/webpack/webpack-cli/blob/617af0bddb5140c07b1a0079b377311b10dd027d/packages/migrate/index.js#L11

Reply:
okay, ,then you should submit a bugfix fixing this to webpack

Reply:
Sorry, I don't understand what we need to fix in ***webpack***?

Should the change be?
```js
`const { validate } = require('webpack'); `
```

Reply:
const { validate } = require('webpack')

Reply:
Yeah. We are using const `WebpackOptionsValidationError` from webpack as well. So,
```js
const { validate, WebpackOptionsValidationError } = require('webpack'); 
```